judgments/tests/test_atom_feed.py

```diff
@@ -112,6 +112,10 @@ def test_feed_entry(self, mock_api_client, mock_search_judgments_and_parse_respo
         assert entry_tna_contenthash is not None
         assert entry_tna_contenthash.text == "ed7002b439e9ac845f22357d822bac1444730fbdb6016d3ec9432297b9ec9f73"
 
+        entry_tna_neutral_citation = entry.find("tna:identifier", self.namespaces)
```

Comment:
Can we get some tests in here to make sure the attributes are as expected as well?